include/boost/python/numpy/config.hpp

```diff
@@ -0,0 +1,79 @@
+//  (C) Copyright Samuli-Petrus Korhonen 2017.
+// Distributed under the Boost Software License, Version 1.0. (See
+// accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+//
+//  The author gratefully acknowleges the support of NMR Solutions, Inc., in
+//  producing this work.
+
+//  Revision History:
+//  15 Feb 17  Initial version
+
+#ifndef CONFIG_NUMPY20170215_H_
+# define CONFIG_NUMPY20170215_H_
+
+# include <boost/config.hpp>
+# include <boost/detail/workaround.hpp>
```

Comment:
It is not used anymore. I just tested it on my build rig. Submitted new version.